dozer-tests/src/e2e_tests/cases/eth/dozer-config.yaml

```diff
@@ -43,3 +43,10 @@ endpoints:
       primary_key:
         - block_number
         - id
+
```

Comment:
Can we avoid having to define udfs in the config, but rather Discovery them from the .py file itself ?

Reply:
If not defined udfs in config, should we require users to put .py file in a specific location?  cc @mediuminvader

Reply:
Would be nice to have a python decorator